source/system/native/MutexUnix.ooc

```diff
@@ -19,7 +19,17 @@ version(unix || apple) {
 			super()
 		}
 		lock: override func {
-			pthread_mutex_lock(this _backend&)
+			version(debugDeadlock) {
+				timeLimit_sec := 1
+				deadline: TimeSpec
+				currentTime: TimeVal
+				gettimeofday(currentTime&, null)
+				(deadline tv_sec, deadline tv_nsec) = (currentTime tv_sec + (timeLimit_sec as TimeT), (currentTime tv_usec * 1000) as Int)
+				if (pthread_mutex_timedlock(this _backend&, deadline&) != 0)
+					raise("Error! [" + this class name + " stalled for more than " + timeLimit_sec + " seconds and timed out]")
```

Comment:
Can't reach Debug class from `MutexUnix`. 

- Import makes the compiler complain as it "leaves" the system folder
- use (some use file with debug) creates a circular dependency as debug is included somewhere else. 
- Also tried creating a small Debug print class inside `MutexUnix` w/o success (as it didn't print anything). 
Are there any other way of reaching `Debug.ooc` as I don't know about?